fs/xfstests.py

```diff
@@ -211,11 +270,11 @@ def setUp(self):
                     self._create_test_list(self.share_exclude, "shared",
                                            dangerous=False)
         if self.detected_distro.name is not 'SuSE':
-            process.run('useradd 123456-fsgqa', sudo=True)
-            process.run('useradd fsgqa', sudo=True)
+            process.run('useradd 123456-fsgqa', sudo=True, ignore_status=True)
```

Comment:
why `ignore_status=True` if user creation fails test should not run right ?

Reply:
Okay, let me add a check to warn if the command fails.